docs/app/components/SampleDetails/SampleDetails.tsx

```diff
@@ -183,7 +149,7 @@ const TemplateDetails: FC<TemplateDetailsProps> = ({
             <Text className={classes.sectionTitle}>Description</Text>
             <div className={classes.contentBox}>
               <div className={classes.description}>
-                {renderMarkdown(longDescription)}
+                {renderMarkdown(longDescription, classes)}
```

Comment:
We need to memoize the result of the remark otherwise it will recompute it on every render.
Is there an option to compute this when we're getting it from the server, so we don't need to do this on the client.

Reply:
It is rendered in build time now